Gemfile

```diff
@@ -2,7 +2,8 @@
 # commit Gemfile and Gemfile.lock.
 source 'https://rubygems.org'
 
-gem 'cocoapods', '1.10.1'
+gem 'cocoapods', '1.11.3'
 
 gem 'cocoapods-generate', '2.0.1'
-gem 'danger', '6.1.0'
+gem 'danger', '8.4.5'
+gem 'xcpretty'
```

Comment:
I updated the gems to match that of the Gemfile in the Firebase repo.

I also added xcpretty as a means to fix the `error: xcpretty command not found` issue. Based on my learnings with the .ruby-version, I suspect that if this is to work, the Gemfile needs to be checked in. I can test and undo this change to the Gemfile/Gemfile.lock if that is not the case. For now, to get this PR's CI green, I have explicitly installed `xcpretty` in the `./setup-scripts.sh` file.